### ISSUE TITLE
Cant use boat

### ISSUE THREAD

**Issue by TheNylox** [NoCheatPlus/Issues#632]
Every player who try to use a boat get freeze after 10 block distance . You can see yourself swimming around the boat but for other players you still in the boat without moving . When happen you cant break your boat "You are not allowed to"

[16:28:58 INFO]: ---- Version information ----
[16:28:58 INFO]: #### Server ####
[16:28:58 INFO]: git-Paper-101 ~MC: 1.16.1~
[16:28:58 INFO]: Detected: 1.16.1
[16:28:58 INFO]: #### NoCheatPlus ####
[16:28:58 INFO]: Plugin: 3.17-SNAPSHOT-b5.9.4
[16:28:58 INFO]: MCAccess: 1.13-1.16.1|? / Bukkit-API
[16:28:58 INFO]: Hooks: ViolationFrequency~NCP~ 1.0
[16:28:58 INFO]: #### Related Plugins ####
[16:28:58 INFO]: CompatNoCheatPlus v6.6.5-RC-sMD5NET-b90 | ProtocolLib v4.6.0 | ConditionalCommands v1.6

Just try to place a boat and use it

mcMMO

No violations in console or in minecraft . They just get freeze . Disabling NCP solve the problem

**Comment by MarkElf** [comment]
Not cool to post my work publicly. Also, can't reproduce any boat-related issues.

**Comment by James-E-A** [comment]
Is this the same issue as #44?

**Comment by MarkElf** [comment]
> Is this the same issue as #44?

No, boats are just janky with 1.16.x. Somewhat solutions have been implemented since.

**Comment by fa0311** [comment]
To solve this problem, change the config.yml as follows
checks.moving.vehicle.envelope.hdistcap.default: 0.9 -> 1.0

**Comment by Janmm14** [comment]
[This project is no longer maintained, go to https://github.com/Updated-NoCheatPlus](https://github.com/NoCheatPlus/Issues/issues/668)
